client/src/components/footer/Footer.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import BottomNavigation from "@material-ui/core/BottomNavigation";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles({
+  root: {
+    height: "10%",
+    width: "100%",
+  },
+});
+
+export default function Footer() {
+  const classes = useStyles();
+
+  return (
+    <BottomNavigation className={classes.root}>
+      <div>
```

Comment:
Try using MUI components instead for consistency (div -> Box, a -> Link, etc). 

Reason is MUI can potentially insert theming on the fly if they're already MUI components.